jujugui/static/gui/src/app/components/inspector/header/header.js

```diff
@@ -21,6 +21,43 @@ class InspectorHeader extends React.Component {
     );
   }
 
+  /**
+    Open getstarted.md of specific charm in post deployment panel
+    @method _navigateToGetStarted
+  */
+  _navigateToGetStarted(e) {
+    e.preventDefault();
+    this.props.changeState(
+      {postDeploymentPanel:
+        {
+          show: true,
+          entityId: this.props.entityId
+        }
+      }
+    );
+  }
+
+  /**
+    Method to navigate to Charm details page from Inpsector
+    @method _navigateToCharmDetails
+  */
+  _navigateToCharmDetails(e) {
+    e.preventDefault();
+    let url;
+    try {
+      url = window.jujulib.URL.fromString(this.props.entityId);
+    } catch (_) {
+      url = window.jujulib.URL.fromLegacyString(this.props.entityId);
+    }
+    this.props.changeState(
+      {
+        profile: null,
```

Comment:
it's unlikely that the inspector will be shown at the same time the profile and search are so I'm not sure it's necessary to null those out. Did you find otherwise?